### ISSUE TITLE
[question] Zoom into tree window

### ISSUE THREAD

**Issue by neerajbadlani** [cs01/gdbgui#178]
##### Describe the issue
Tree Window not big enough to get useful insights for bug projects , is it possible to
open tree window in seperate browser window .

**Comment by cs01** [comment]
Have you tried inputting pixel dimensions (at the top of the tree component) and pressing enter? You can resize the area that the tree is rendered in. If that is not enough I can look into a new window.

**Comment by neerajbadlani** [comment]
Yeah , I did try , but even then breaking it into its window for complex data structures would be awesome . Also , if somehow it can be made more responsive , that would be great . For me tree  is 'THE' item , I tried my hand couple of years back to get tree output , but it was just static graph files pieced together

**Comment by cs01** [comment]
Got it. I will put it on my todo list to allow it to open in a new window.

**Comment by yimyom** [comment]
any progress on this feature? It would be the TOP feature of this software if it can be used in a separate window.

**Comment by cs01** [comment]
Thanks for the ping. Unfortunately there has not been any progress. If I get back to working on gdbgui I can put this at the top of the list, but I don't know if/when that will happen as this is a project I do for fun in my spare time.

**Comment by jboockmann** [comment]
The module [`react-new-window`](https://www.npmjs.com/package/react-new-window) looks quite promising for this task.
